app/src/main/java/com/rebalance/utils/UtilsUI.kt

```diff
@@ -13,3 +31,59 @@ fun alertUser(message: String, context: Context) {
         ).show()
     }
 }
+
+@Composable
+fun displayExpenseImage(
```

Comment:
Please move it to ui/components/main package as separate component.

Reply:
done